spt/strafestuff.cpp

```diff
@@ -265,7 +265,23 @@ bool Strafe(PlayerData& player, const MovementVars& vars, bool onground, bool ju
 			out.Yaw = NormalizeRad(atan2(vel.y, vel.x));
 		}
 		else {
+			#ifndef OE && P2
```

Comment:
I believe in the rest of the code preprocessor commands are not indented. Please make this consistent with the rest of the code. Also I'm pretty sure you can't use `&&` in `#ifndef`. Did you test if this compiles on other SDKs?